package.json

```diff
@@ -120,7 +120,7 @@
     "parse-dashboard": "./bin/parse-dashboard"
   },
   "engines": {
-    "node": ">=4.3"
+    "node": ">=8.9"
```

Comment:
Is this required?

Reply:
We have some dependencies that require at least node 8.x, as [Parse-Server](https://github.com/parse-community/parse-server/blob/master/package.json#L109). Besides that, sass-loader announced a breaking change in the [changelog](https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#800-2019-08-29) about minimum required node version.

Reply:
I think it is pretty reasonable to require at least node >= 8.9 as in Parse Server. @dplewis @acinader thoughts?

Reply:
Yes, I'm good with it.  Node 8 is at 8.16 and it goes out of maintenance in a few months at which point I think we set the minimum at 10.

More broadly (and to invite criticism), I think for node version for all concerned repositories (and parse-dashboard in particular), we should not bend over backward to support obsolete versions like early node 8 versions.

Users don't have to upgrade to newer versions of parse if they are unwilling to keep up with dependencies.

Reply:
I agree. Let's go to Parse Dashboard 2.0 then.